### PR TITLE
Set logging channel as stable for version >= OCP 4.7

### LINKED ISSUE #3986: Set logging channel as "Stable" for versions >= OCP 4.7
No need to skip logging versions, from OCP 4.7 logging has become a standalone product.

for versions lesser than 4.7 its important to mention the right channels, But from 4.7 and above just mentioning the channel as "stable" should work.

### PR DESCRIPTION
**Pull request by Shrivaibavi**
Fixes #3986

Signed-off-by: Shrivaibavi Raghaventhiran <sraghave@redhat.com>

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -2343,8 +2343,9 @@ def finalizer():
 
     # Checks OCP version
     ocp_version = get_running_ocp_version()
+    ocp_version = ["stable" if ocp_version >= "4.7" else ocp_version]
```
Comment (clacroix12) [inline tests/conftest.py:2346]: You can do this without the list comprehension to keep `ocp_version` a string. This way you don't need to use `ocp_version[0]` below.

```suggestion
    ocp_version = "stable" if ocp_version >= "4.7" else ocp_version
```

Comment (Shrivaibavi) [reply tests/conftest.py]: Done

**Comment by Shrivaibavi** [comment]
PR validation passed:
https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/1343/

File: tests/conftest.py [hunk 2]
```diff
@@ -2343,8 +2343,9 @@ def finalizer():
 
     # Checks OCP version
     ocp_version = get_running_ocp_version()
+    ocp_version = "stable" if ocp_version >= "4.7" else ocp_version
```
Comment (petr-balogh) [inline tests/conftest.py:2346]: I would rename this variable to logging_channel

Comment (Shrivaibavi) [reply tests/conftest.py]: done

**Review by petr-balogh — APPROVED** [review]

**Review by Akarsha-rai — APPROVED** [review]

**Pull request merged**